purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -132,21 +132,21 @@ fun Purchases.purchaseProductWith(
 }
 
 /**
- * Purchase product.
+ * Purchase a [Product]'s [PurchaseOption].
  * @param [activity] Current activity
  * @param [storeProduct] The storeProduct of the product you wish to purchase
  * @param [purchaseOption] Your choice of purchase options available for the StoreProduct
  * @param [onSuccess] Will be called after the purchase has completed
  * @param [onError] Will be called after the purchase has completed with error
  */
-fun Purchases.purchaseProductWithOption(
+fun Purchases.purchaseProductOptionWith(
```

Comment:
I don't recall why they had to be named different from the Java versions, and why we added the `with` at the end of these functions, but this makes it more consistent with the naming in this file, so good catch